vendors/circom/prover_main.cc

```diff
@@ -149,26 +141,29 @@ TimeInfo CreateProof(const base::FilePath& zkey_path,
     }
 
     end = base::TimeTicks::Now();
-    base::TimeDelta prove = end - start;
-    time_info.AddProve(prove);
+    base::TimeDelta delta = end - start;
+    std::cout << "Time taken for proving #" << i << ": " << delta << std::endl;
+    prover_time.Add(delta);
```

Comment:
Is the prover_time total/max used anywhere?

Reply:
Oh I forgot to add this. Thank you for pointing out this.